Objects/longobject.c

```diff
@@ -2820,6 +2820,59 @@ that triggers it(!).  Instead the code was tested by artificially allocating
 just 1 digit at the start, so that the copying code was exercised for every
 digit beyond the first.
 ***/
+
+// Tables are computed by Tools/scripts/long_conv_tables.py
+#if PYLONG_BITS_IN_DIGIT == 15
+    static double log_base_BASE[37] = {0.0, 0.0, 0.06666666666666667,
+        0.10566416671474375, 0.13333333333333333, 0.15479520632582416,
+        0.17233083338141042, 0.18715699480384027, 0.19999999999999998,
+        0.2113283334294875, 0.22146187299249084, 0.23062877457581984,
+        0.2389975000480771, 0.24669598120940617, 0.25382366147050694,
+        0.26045937304056793, 0.26666666666666666, 0.27249752275002265,
+        0.27799500009615413, 0.2831951675629057, 0.28812853965915747,
+        0.29282116151858406, 0.2972954412424865, 0.3015707970704675,
+        0.3056641667147438, 0.30959041265164833, 0.3133626478760728,
+        0.31699250014423125, 0.3204903281371736, 0.3238653996751715,
+        0.3271260397072346, 0.3302797540257917, 0.33333333333333337,
+        0.3362929412905636, 0.3391641894166893, 0.34195220112966446,
+        0.34466166676282084};
+    static int convmultmax_base[37] = {0, 0, 32768, 19683, 16384,
+        15625, 7776, 16807, 32768, 6561, 10000, 14641, 20736, 28561,
+        2744, 3375, 4096, 4913, 5832, 6859, 8000, 9261, 10648, 12167,
+        13824, 15625, 17576, 19683, 21952, 24389, 27000, 29791, 32768,
+        1089, 1156, 1225, 1296};
+    static twodigits convwidth_base[37] = {0, 0, 15, 9, 7, 6, 5, 5, 5,
```

Comment:
`convwidth_base` should be `int`

Reply:
Ah, good catch.  I swapped the array order in my Python script but not the types.  Fixed.